src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -486,14 +486,14 @@ private Size ComputeLargeContentSize()
 
             var largeContentGridSize = largeContentGrid.DesiredSize;
 
-            // Add two times width of scroll bar (5) and right margin(5), refer to infoBubbleView.xaml
-            largeContentGridSize.Width += 20;
-
             // Don't make it smaller then min width.
             largeContentGridSize.Width = largeContentGridSize.Width < largeContentGrid.MinWidth
                 ? largeContentGrid.MinWidth
                 : largeContentGridSize.Width;
 
```

Comment:
In very few cases the default maximum 280 is less than the minWidth of host, like colorRange node, so the largeContentGridSize will be adjusted again here. Change the code sequence to reflect that.